job_definitions/search_environment.yml

```diff
@@ -0,0 +1 @@
+{% set environments = [('preview', 'g-cloud-9', 'g-cloud-9'), ('staging', 'g-cloud-9', 'g-cloud-9'), ('production', 'g-cloud-9', 'g-cloud-9')] %}
```

Comment:
If you just want to define a variable you can add this to `playbooks/roles/jenkins/defaults/main.yml` as that's the main vars file we use for non-secret stuff. And maybe make this a list of dictionaries.